frontend/src/app/(routes)/governance/ProposalOverviewDeposit.tsx

```diff
@@ -0,0 +1,183 @@
+'use client';
+import React, { useState } from 'react';
+import Image from 'next/image';
+import CustomPieChart from './CustomPiechart';
+import './style.css';
+import ProposalDetailsCard from './ProposalDetailsCard';
+import DepositPopup from './DepositPopup';
+
+const ProposalOverviewDeposit = ({
+  proposalId,
+  proposalText,
+  proposalname,
+}: {
+  proposalId: string;
+  proposalText: string;
+  proposalname: string;
+}) => {
+  const [isDepositPopupOpen, setDepositPopupOpen] = useState(false);
+  const toggleDepositPopup = () => {
+    setDepositPopupOpen(!isDepositPopupOpen);
+  };
+  const data = [
+    { value: 75, color: '#4AA29C', label: 'Yes' },
+    { value: 23, color: '#E57575', label: 'No' },
+    { value: 2, color: '#EFFF34', label: 'Veto' },
+    { value: 0, color: '#EFFF34', label: 'Veto' },
+  ];
+  const dataset = [
+    { value: 75, color: '#759BE5', label: 'Quorum' },
+    { value: 23, color: '#75E5A2', label: 'Turn out' },
+    { value: 2, color: '#B373CA', label: 'Threhold' },
+  ];
+  const quorum = 50;
```

Comment:
it is not good practice to define default values like this. take the above advice.